### DISCUSSION TITLE
why I can not move pic from tag 1.10.2?

### DISCUSSION THREAD

**Discussion by oasahi** [fengyuanchen/viewerjs#599] [Q&A]
In tag 1.10.1 event bind,here is "document = this.viewer.ownerDocument" ,and in tag 1.10.2 here is "document = this.element.ownerDocument".
Why?

**Comment by fengyuanchen** [comment]
How about the latest tag v1.11.3?

**Reply by oasahi** [reply]
The lastest tage is same as tag 1.10.2,no change in code "document = this.element.ownerDocument",it is no work.
It is "document = this.viewer.ownerDocument" in move event bind function before tag 1.10.1,and it's work fine.

**Reply by fengyuanchen** [reply]
Cannot find the code you referenced in v1.10.1.
